mobly/base_test.py

```diff
@@ -838,21 +838,22 @@ def exec_one_test(self, test_name, test_method, record=None):
         self.current_test_info = None
     return tr_record
 
-  def _assert_function_name_in_stack(self, expected_func_name):
-    """Asserts that the current stack contains the given function name."""
+  def _assert_function_names_in_stack(self, expected_func_names):
+    """Asserts that the current stack contains any of the given function names.
+    """
     current_frame = inspect.currentframe()
     caller_frames = inspect.getouterframes(current_frame, 2)
     for caller_frame in caller_frames[2:]:
-      if caller_frame[3] == expected_func_name:
+      if caller_frame[3] in expected_func_names:
```

Comment:
Sorry, why is this file modified again? I think the modification is merged into master.

This PR should only modify 2 files, but I see this PR modified 4 files, which is strange. Is this because I'm viewing the diff in the wrong way?

Reply:
Resolving.

This is just because I'm viewing the diff in the wrong way.